app/javascript/components/subject/MonitoringActions.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Form, Row, Col } from 'react-bootstrap';
+
+import AssignedUser from './AssignedUser';
+import CaseStatus from './CaseStatus';
+import Jurisdiction from './Jurisdiction';
+import MonitoringStatus from './MonitoringStatus';
+import GenericAction from './GenericAction';
+
+class MonitoringActions extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <Form className="mb-3 mt-3 px-4">
+          <Row>
```

Comment:
Oh another comment I forgot to add, I don't think this `Row` or the `Col` are necessary here.

Reply:
done